apps/api/.env.example

```diff
@@ -3,3 +3,4 @@ JWT_SECRET=
 MOCKS=false
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
+ENC_SECRET=
```

Comment:
What should this value be for a local & for a production environment? Any secret-type string? I also think that `ENCRYPTION_SECRET` is a slightly better name.

Reply:
According to the [documentation](https://www.npmjs.com/package/mongoose-encryption), the secret should be a 32/64 bit base64 string.